modules/simple_forms_api/app/form_mappings/vba_20_10207.json.erb

```diff
@@ -49,23 +50,22 @@
   "form1[0].#subform[3].Email_Address[3]": "<%= nil %>",
   "form1[0].#subform[3].VA_File_Number_If_Applicable[1]": "<%= nil %>",
 
-  "form1[0].#subform[3].InsuranceNumber[0]": "<%= nil %>",
-  "form1[0].#subform[3].CurrentlyHomeless[0]": "<%= nil %>",
-  "form1[0].#subform[3].HomelessLivingSituation[0]": "<%= nil %>",
-  "form1[0].#subform[3].Other1[0]": "<%= form.data['other_housing_risks'] ? 1 : 0 %>",
-  "form1[0].#subform[4].Veterans_SocialSecurityNumber_LastFourNumbers[1]": "<%= form.data.dig('id', 'ssn')&.[](5..8) %>",
-  "form1[0].#subform[4].Veterans_SocialSecurityNumber_SecondTwoNumbers[1]": "<%= form.data.dig('id', 'ssn')&.[](3..4) %>",
-  "form1[0].#subform[4].Veterans_SocialSecurityNumber_FirstThreeNumbers[1]": "<%= form.data.dig('id', 'ssn')&.[](0..2) %>",
-  "form1[0].#subform[4].RadioButtonList[0]": "<%= nil %>",
-  "form1[0].#subform[4].RadioButtonList[1]": "<%= nil %>",
+  "form1[0].#subform[3].CurrentlyHomeless[0]": "<%= form.currently_homeless? ? 0 : 1 %>",
+  "form1[0].#subform[3].HomelessLivingSituation[0]": "<%= form.homeless_living_situation %>",
+  "form1[0].#subform[3].Other1[0]": "<%= nil %>",
```

Comment:
Are we no longer using `form.data['other_housing_risks']` here?

Reply:
Yeah this is a strange one. Both subsections have an Other box but we're only filling one of them in (the other one) since we're only collecting a free-entry Other input once on the FE. I guess we could replicate it in both boxes on the PDF but my inclination was to leave one blank and fill the other. Do you think it's better to have the same content in both spots?

Reply:
Honestly, if the form is accepted as-is by its final destination, I have no issues with it.